server/form-pages/apply/risks-and-needs/health-needs/guidance.ts

```diff
@@ -0,0 +1,43 @@
+import type { TaskListErrors } from '@approved-premises/ui'
+import { Cas2Application as Application } from '@approved-premises/api'
+import { Page } from '../../../utils/decorators'
+import TaskListPage from '../../../taskListPage'
+
+type GuidanceBody = Record<string, never>
```

Comment:
oh cool, I haven't come across this type before - is it basically a way of saying that the `body` will be an empty object?

Reply:
yes: I started off by trying to define the type with:

```ts
type GuidanceBody = {}
```

EsLint then told me:

```
Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `object` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead.
- If you really want a type meaning "any non-nullish value", you probably want `NonNullable<unknown>` instead.
```